### ISSUE TITLE
Add udev rules for devices

### ISSUE THREAD

**Issue by noonsleeper** [flathub/net.displaycal.DisplayCAL#2]
Hey, I tried to use this flatpak and never detect my `Spyder X`, then after a research I found that we need to create an udev rule that allow access to the specific devices.

My question is, Is it good if we add to this repo a directory where we can put all this udev rules and a simple explanation in a readme in the same directory to explain how to install in the OS? (I can create a PR with this and the simple rule that I create for the `Spyder X` as example)

**Comment by hfiguiere** [comment]
Flatpak has no support for udev rules.

**Comment by noonsleeper** [comment]
Yes, I know that doesn't support per se, but you need that in the host to make access to the usb devices, then I think is better if the instructions and the rules live in this repo
